maven-failsafe-plugin/pom.xml

```diff
@@ -45,6 +45,13 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+      <version>1.7.36</version>
+      <scope>test</scope>
```

Comment:
We declare `slf4j-simple` in scope test but we used `slf4j-api` from transitive dependencies in production code. 

I would like to add also `slf4j-api` in standard compile scope.
W can also add version in root pom in dependencies management instead of define version in each module.

Reply:
Ah yes, should have done that in the first place, thank you.  Done.